src/components/TopBar/logo.js

```diff
@@ -1,13 +1,15 @@
 import React from "react";
-import logo from "../../assets/icons/msm_logo.svg";
+import logo from "../../assets/icons/msm_logo.png";
 
 export default class Logo extends React.Component {
   render() {
     return (
         <div id="logo">
-            <img src={logo}
-            alt="Migrate Service Map"
-            />
+            <a href="https://refugeeswelcomehome.org/">
```

Comment:
Could you add target="_blank" to this element? That make the link open in a new tab. The map does not store saved items between page loads so we don't want to lose progress if the user clicks on the logo.

Reply:
@alexjball This has been added in commit #8c14aab